tensorflow/c/eager/BUILD

```diff
@@ -256,6 +256,73 @@ tf_cuda_cc_test(
     ],
 )
 
+cc_library(
+    name = "mnist_gradients_testutil",
+    srcs = [
+        "mnist_gradients_testutil.cc",
+        "mnist_gradients_testutil.h",
+    ],
+    hdrs = [
+        "gradients.h",
+    ],
+    visibility = [
+        "//tensorflow:internal",
+    ],
+    deps = [
+        ":abstract_context",
+        ":abstract_operation",
+        ":abstract_tensor_handle",
+        ":c_api_unified_internal",
+        ":gradients_internal",
+        ":tape",
+        "//tensorflow/c/experimental/ops:array_ops",
+        "//tensorflow/c/experimental/ops:math_ops",
+        "//tensorflow/core/common_runtime/eager:attr_builder",
+        "//tensorflow/core/lib/llvm_rtti",
+        "//tensorflow/c/experimental/ops:nn_ops",
```

Comment:
Buildifier still complains. I think moving `"//tensorflow/c/experimental/ops:nn_ops",` to line 280 should fix it.